ansible/roles/vm_set/tasks/control_mux_simulator.yml

```diff
@@ -7,6 +7,14 @@
     become: yes
     environment: "{{ proxy_env | default({}) }}"
 
+  - name: Get absolute path of {{ root_path }}
+    command: "realpath {{ root_path }}"
+    register: real_root_path
+
+  - name: Set {{ root_path }} to absolute path
+    set_fact:
+      root_path: "{{ real_root_path.stdout }}"
```

Comment:
I suggest to introduce a new var instead of ```root_path``` because other tasks may still treat it as a relative path.

Reply:
Updated. Thanks!